lib/timers.js

```diff
@@ -154,7 +155,7 @@ function setTimeout(callback, after, arg1, arg2, arg3) {
       args = [arg1, arg2, arg3];
       for (i = 5; i < arguments.length; i++) {
         // Extend array dynamically, makes .apply run much faster in v6.0.0
-        args[i - 2] = arguments[i];
+        ArrayPrototypePush(args, arguments[i]);
```

Comment:
I suspect they will all be similar

The goal here is to minimize HOLEY internal arrays and not press on objects too much, [it might hurt other parts of V8](https://stackoverflow.com/questions/62350146/why-map-manipulation-is-much-slower-than-object-in-javascript-v8-for-integ/62351925#62351925)